### PR TITLE
7701 fix blank currencies for fake data

### PR DESCRIPTION
**Pull request by arisah-ijeoma**

### CODE REVIEW

File: spec/factories/organizations.rb [hunk 1]
```diff
@@ -45,7 +45,6 @@
 FactoryBot.define do
   factory :organization do
     division { root_division }
-    country
```
Comment (emitche) [inline spec/factories/organizations.rb]: Why did you remove country from the factory?

Comment (arisah-ijeoma) [reply spec/factories/organizations.rb]: it was responsible for adding new currencies since there's an association

Comment (emitche) [reply spec/factories/organizations.rb]: Ok. Thanks.
